sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/aio/_document_model_administration_client_async.py

```diff
@@ -113,6 +114,8 @@ async def begin_build_model(
         :param str source: An Azure Storage blob container's SAS URI. A container URI (without SAS)
             can be used if the container is public. For more information on setting up a training data set, see:
             https://aka.ms/azsdk/formrecognizer/buildtrainingset
+        :param build_mode: The custom model build mode. Possible values include: "template", "neural".
```

Comment:
We'll definitely need to link to some documentation on this once it exists. 😄

Reply:
Will do!